tutel/experts/ffn.py

```diff
@@ -101,10 +101,10 @@ def forward(self, x, ctx):
 
     def to(self, *args, **kwargs):
         self = super().to(*args, **kwargs)
-        self.fc1_weight = self.fc1_weight.to(*args, **kwargs)
-        self.fc2_weight = self.fc2_weight.to(*args, **kwargs)
-        self.fc1_bias = self.fc1_bias.to(*args, **kwargs)
-        self.fc2_bias = self.fc2_bias.to(*args, **kwargs)
+        self.batched_fc1_w = self.batched_fc1_w.to(*args, **kwargs)
+        self.batched_fc2_w = self.batched_fc2_w.to(*args, **kwargs)
+        self.batched_fc1_bias = self.batched_fc1_bias.to(*args, **kwargs)
+        self.batched_fc2_bias = self.batched_fc2_bias.to(*args, **kwargs)
         return self
```

Comment:
Thanks, it makes sense.

Reply:
this fn should actually probably just be removed since the movement of these parameters is already accomplished by the previous call to `super().to(*args, **kwargs)`. But if all you're doing in an fn is calling super(), why have it at all?